07/ID_funkce/datum_narozeni.py

```diff
@@ -0,0 +1,16 @@
+def birthdate(ID):
+
+    year = int(ID[:2])
+    if year > 85 and year < 100:
+        year = 1900 + int(year)
+    elif year < 24:
+        year = 2000 + int(year)
+
+    if int(ID[2:4]) > 50:
```

Comment:
Tu by som nahradila celý ten if len za month = int(ID[2:4]) % 50

Nie je nutné prehadzovať naspäť na str, to si urobí formát na r. 16 sám.